### ISSUE TITLE
Multi-sample TITAN

### ISSUE THREAD

**Issue by fpbarthel** [gavinha/TitanCNA#58]
Hi Gavin,

Your paper that you shared in a previous post (https://www.ncbi.nlm.nih.gov/pubmed/29909985) uses PyClone on top of TITAN, using the TITAN major and minor copy number (I guess) to provide the correct input to PyClone. I'm wondering why did you chose this approach?

For my data, I am interested in comparing clonal clusters across multiple related samples. TITAN I guess is a single-sample approach: "cluster 2" in two related samples may not refer to the same cluster. PyClone on the other hand can take a multi-sample approach, and would be able to track the same clusters between multiple related samples. Is this a sound strategy? Do you have any special suggestions for using TITAN output as PyClone input?

Thanks!
Floris

**Comment by gavinha** [comment]
Hi @fpbarthel 


> Your paper that you shared in a previous post (https://www.ncbi.nlm.nih.gov/pubmed/29909985) uses PyClone on top of TITAN, using the TITAN major and minor copy number (I guess) to provide the correct input to PyClone. I'm wondering why did you chose this approach?

We didn't use PyClone for the prostate cancer study, but the approach of using TITAN major/minor CN as input is something we've used a lot before. TITAN and PyClone were developed in the same lab so it was convenient. However, other tools that generate major/minor CN can also be used.

> For my data, I am interested in comparing clonal clusters across multiple related samples. TITAN I guess is a single-sample approach: "cluster 2" in two related samples may not refer to the same cluster. PyClone on the other hand can take a multi-sample approach, and would be able to track the same clusters between multiple related samples.

Yes, TITAN is a single sample approach and your comment is correct. 

> Is this a sound strategy? 

Here, you just need to be conservative about your interpretations on the predicted clonal structure between TITAN and PyClone results. One limitation here is for overlapping CNAs and SNVs and need to model the ordering of events in the context of a phylogeny. There are other tools that take cellular prevalence estimates from PyClone and constructs tree phylogenies from multiple samples (e.g. LICHeE, ClonEvol). 

> Do you have any special suggestions for using TITAN output as PyClone input?

In the past, I have tried using some simple approaches to perform multi-dimensional clustering (e.g. k-means) between the clones of multiple samples after running TITAN. It's not the best approach, but gives something that relates the samples. While I have considered extending the HMM to consider multiple samples, I never got around to it. 
There are other tools, such as cloneHD, that do support multi-sample analysis. 

Best,
Gavin
